sonaranalyzer-dotnet/CBDE/LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+Copyright (C) 2012-2019 SonarSource SA
```

Comment:
This was extracted from initial zip file: [cbde-windows.1.0.0.14843.zip](https://repox.jfrog.io/repox/webapp/#/artifacts/browse/tree/General/sonarsource-public-releases/org/sonarsource/cbde/1.0.0.14843/cbde-windows.1.0.0.14843.zip)

Reply:
> I am trying to find what script we use to update the copyright details, this should be added there as well, but cannot find any resource...

I don't remember to use a script for this and I didn't know we have one. I'm not sure what to regarding this.

> we have this page but I believe it refers only to cs/java headers.

I've added also a section for the CBDE related files.